exec/java-exec/src/main/java/org/apache/drill/exec/dotdrill/DotDrillUtil.java

```diff
@@ -30,6 +32,9 @@
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DotDrillUtil.class);
 
   private static List<DotDrillFile> getDrillFiles(DrillFileSystem fs, FileStatus[] statuses, DotDrillType... types){
+    if (statuses == null) {
+      return Collections.emptyList();
+    }
```

Comment:
Is this a legal case? Should this, instead, be:

```
if (statuses == null) { throw new IllegalArgumentException( ... ); }
```

(Or the `Preconditions` equivalent...)